### ISSUE TITLE
stdout/stderr output of actions not logged

### ISSUE THREAD

**Issue by MaZderMind** [bundlewrap/bundlewrap#290]
> As a bundle-developer I want to see the output of an `actions[…]` command, when debugging is enabled

For example an action specified as
```
    actions[name] = {
        'command': '/usr/local/bin/wp core config --path=/vagrant/wordpress/htdocs --dbname=wordpress --dbuser=wordpress --dbpass=wordpress',
        'needs': ['symlink:/usr/local/bin/wp'],
    }
```

will not show the output of `command` anywhere in the otherwise very verbose debug-output. The actual output looks something like this:
```
[2016-11-21 17:45:12.858825] [DEBUG] running: ssh -o KbdInteractiveAuthentication=no -o PasswordAuthentication=no -o StrictHostKeyChecking=no customer-webseite sudo sh -c 'export LANG=C; /usr/local/bin/wp core config --path=/vagrant/wordpress/htdocs --dbname=wordpress --dbuser=wordpress --dbpass=wordpress'
…
[2016-11-21 17:45:13.071995] [DEBUG] command finished with return code 1
[2016-11-21 17:45:13.073533] [DEBUG] worker pool apply_customer-webseite delivering result of customer-webseite:action:wordpress-core-config on worker #2
[2016-11-21 17:45:13.074648] [DEBUG] skipped these items because they depend on action:wordpress-core-config, which was skipped previously: action:, bundle:wordpress
[2016-11-21 17:45:13.075313] [DEBUG] skipped these items because they depend on bundle:wordpress, which was skipped previously: bundle:
[2016-11-21 17:45:13.075574] ✘ customer-webseite  wordpress  action:wordpress-core-config failed 
```

This caught us especially with the wp-cli command that refused to run as root and reported so on stderr while returning 1 as return-code.
Just from copy'n'pasting the Commands from the Debug-Output the underlying problem was not visible and the only way to debug this kind of issue is currently piping the command-output into a file on the target-system.

It would be good to see all relevant output in the debug-log.

**Comment by trehn** [comment]
+1, but only if the action fails

while we're at it, we can do this for any other command that fails

**Comment by trehn** [comment]
fixed in 7845e54e
